components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -97,13 +100,18 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         this.props.actions.getProfilesNotInChannel(this.props.channel.team_id, this.props.channel.id, this.props.channel.group_constrained, 0).then(() => {
             this.setUsersLoadingState(false);
         });
+        this.props.actions.getProfilesInChannel(this.props.channel.id, 0, USERS_PER_PAGE, '', true);
         this.props.actions.getTeamStats(this.props.channel.team_id);
         this.props.actions.loadStatusesForProfilesList(this.props.profilesNotInCurrentChannel);
+        this.props.actions.loadStatusesForProfilesList(this.props.profilesInCurrentChannel);
```

Comment:
Can we concat the 2 arrays (this.props.profilesInCurrentChannel & this.props.profilesNotInCurrentChannel) and make this into one request?